orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationConfig.java

```diff
@@ -0,0 +1,44 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.core.AiModel;
+import io.vavr.control.Option;
+import javax.annotation.Nonnull;
+
+/**
+ * Represents the configuration for the orchestration service. Allows for configuring the different
+ * modules of the orchestration service via a fluent API.
+ *
+ * @param <T> Type of the specific implementation to make a fluent API possible.
+ */
+public interface OrchestrationConfig<T extends OrchestrationConfig<T>> {
+
+  @Nonnull
+  Option<AiModel> getLlmConfig();
```

Comment:
Debatable, if we want to keep VAVR in our public API. Done here for now because this makes [our own code](https://github.com/SAP/ai-sdk-java/pull/114/files?diff=unified&w=0#diff-a3400feb9c493fbbc88bb89a699010093d0cb6a3364e016004ee417695746ff4R116) (arguably) much more beautiful 😄 

Alternatives would be Java `Optional` or just `@Nullable`.